rendermodules/em_montage_qc/detect_montage_defects.py

```diff
@@ -146,7 +146,8 @@ def detect_stitching_gaps(render, prestitched_stack, poststitched_stack,
     gap_tiles = []
     for n in G2.nodes():
         if G1.degree(n) > G2.degree(n):
-            tileId = pre_tileIds.keys()[pre_tileIds.values().index(n)]
+            # FIXME is this correct for python2?
```

Comment:
I think this is okay...

Reply:
Right , it looks like nothing happens to that dictionary after creation that would change the order.  Will remove comment,